rskj-core/src/main/java/co/rsk/rpc/ExecutionBlockRetriever.java

```diff
@@ -82,7 +88,7 @@ public Block getExecutionBlock(String bnOrId) {
         }
 
         if (executionBlockNumber.isPresent()) {
-            Block executionBlock = miningMainchainView.getBlockByNumber(executionBlockNumber.get());
+            Block executionBlock = blockchain.getBlockByNumber(executionBlockNumber.get());
```

Comment:
what was the motivation for replacing miningMainchainView with blockchain?

Reply:
Because `ExecutionBlockRetriever` uses Blocks and now `MiningMainchainView` stores `BlockHeaders`. 
In the future, this class will be refactored to use either the `BlockStore` or a specific view. 
We thought it would be better to not add that refactor to RSKIP 110 feature which is already adding several code changes.